demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -70,7 +70,7 @@ def build_argparser():
                       help='Optional. Background blur strength (by default with value 0 is not applied).')
     args.add_argument('--layout', type=str, default=None,
                       help='Optional. Model inputs layouts. '
-                           'Format "[<layout>]" or "<input1>[<layout1>],<input2>[<layout2>]" in case of more than one input.'
+                           'Ex. NCHW or input0:NCHW,input1:NC in case of more than one input.'
                            'To define layout you should use only capital letters')
```

Comment:
There is no such note for C++. `-d` and `--streams` require the same syntax, but they don't have such note either. I think `To define layout you should use only capital letters` should be removed

Reply:
Removed.